packages/opencensus-core/test/test-root-span.ts

```diff
@@ -78,13 +76,51 @@ describe('RootSpan', () => {
     });
   });
 
+  describe('nested spans', () => {
+    it('should get nested spans from rootspan instance', () => {
+      const root = new RootSpan(tracer, name, kind, traceId, parentSpanId);
+      root.start();
+      assert.equal(root.numberOfChildren, 0);
+      const child1 = root.startChildSpan('child1', types.SpanKind.UNSPECIFIED);
+      assert.equal(root.numberOfChildren, 1);
+      assert.equal(child1.numberOfChildren, 0);
+      const child2 = root.startChildSpan('child2', types.SpanKind.UNSPECIFIED);
+      assert.equal(root.numberOfChildren, 2);
+      const grandchild1 = child1.startChildSpan({
```

Comment:
Can you please add tests for same using top level `tracer` object in `test-tracer.ts` ?

Reply:
Added tests to test-tracer.ts.

Reply:
Thanks. This is what I wanted to see, looks like with current implementation you need to do `child1.startChildSpan` to create a grandchild. AFAIK we should always use `tracer` instance to create a span(s). What do you think about passing an additional (optional) option (something like `childOf`) to the `startChildSpan` function to achieve the same?

Example : 
```
tracer.startChildSpan(
      name: 'grandchild1',
      kind: types.SpanKind.UNSPECIFIED, 
      childOf: child1
);
```

Reply:
Done